docs/agents/index.md

```diff
@@ -3,23 +3,25 @@ description: Sematext has several different Agents available for you to monitor
 
 To start monitoring your infrastructure you need to set up the
 appropriate Sematext monitoring agent. You can choose from:
+  
+  - [Sematext Agent](../agents/sematext-agent/), a lightweight, blazing 
+    fast Go-based Monitoring Agent with a tiny footprint for both 
+    infrastructure and containers.
 
   - [Sematext Infra & App Agent](spm-client) (aka SPM Client or SPM
     Monitor). The Infra Agent collects OS & Network metrics.
-    It also collects information about processes and infrastructure inventory.
     The App Agent collects application metrics. The App Agent
     can run in [embedded](spm-monitor-javaagent)
     (aka Java agent-based, aka in-process) or
     [standalone](spm-monitor-standalone) mode.  This agent can also
     instrument JVM-based apps to collect transaction traces and
     perform [on demand profiling](on-demand-profiling).
 
-  - [Node-based App Agent](node-agent), which can [monitor
+  - [Node.js-based App Agent](node-agent), which can [monitor
     Apache](../integration/apache) and [Nginx](../integration/nginx)
     (including [Nginx Plus](../integration/nginxplus)).
 
-  - [Docker Agent](../containers), which can collect not
+  - [Sematext Docker Agent (Deprecated)](../sematext-docker-agent), which can collect not
```

Comment:
Do we really want to link to something that's deprecated?

Reply:
Removed the link.